.github/workflows/test.yml

```diff
@@ -15,6 +15,9 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DEPLOYER_IMAGE: quay.io/domino/deployer:develop.f72b81d5db7e04cc48478d310eafa4abb927ce7f
```

Comment:
I missed this originally, it's no longer used and the associated step is no longer useful yeah?

Reply:
Yeah, I nuked it it.